react/components/LocaleSwitcher/index.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+import GlobeIcon from './GlobeIcon'
+
+const LocaleSwitcher = () => {
+  const supportedLangs = [
+    {
+      text: 'EN',
+      id: 'en-US',
+    },
+    {
+      text: 'PT',
+      id: 'pt-BR',
+    },
+    {
+      text: 'ES',
+      id: 'es-AR',
+    },
+  ]
+  const { culture, emitter } = useRuntime()
+  const [openLocaleSelector, setOpenLocaleSelector] = useState(false)
+  const [selectedLocale, setSelectedLocale] = useState(
+    findLocale(culture.locale)
+  )
+
+  const handleLocaleClick = ({ target: { id } }) => {
+    emitter.emit('localesChanged', id)
+    setOpenLocaleSelector(false)
+    setSelectedLocale(findLocale(id))
+  }
+
+  const handleMouseDown = e => {
+    e.preventDefault()
```

Comment:
why do you need this?

Reply:
if don't put `preventDefault`, when clicking on another language it will not be selected